src/components/media/Tag.tsx

```diff
@@ -27,7 +30,8 @@ export default function Tag ({ tag, onDelete, size = 'md', showDelete = false, i
         onClick={stopPropagation}
       >
         {isArea && <MapPinIcon className='w-4 h-4' />}
-        <div className='mt-0.5 whitespace-nowrap truncate text-sm'>{name}</div>
+
+        <div style={maxWidth !== undefined ? { maxWidth: maxWidth } : { maxWidth: DEFAULT_TAG_WIDTH }} className='mt-0.5 whitespace-nowrap truncate text-sm'>{name}</div>
```

Comment:
what do you think if we just default max width to Tailwind `max-width-xs` which is 320?  And set `title` attribute to `name` to show a tool tip on desktop?  This way we don't have to introduce the maxWidth param?
```html
<div className='mt-0.5 whitespace-nowrap truncate text-sm max-w-xs ' title={name}>{name}</div>
```

https://tailwindcss.com/docs/max-width

Reply:
I looked into using that class an it is not narrow enough for some screen sizes and the small cards. 240px or so is the max width for several screen sizes and layouts.  See Screen Shot.

Maybe it would be better to limit the tag length on the admin side?  Or alternatively write a small utility function that truncates long tags?

![Screen Shot 2023-02-08 at 7 22 07 PM](https://user-images.githubusercontent.com/24211/217701315-6bd51143-6e0b-4d11-a083-5677f7a69c3f.png)

Reply:
Ah, you're right. In which case we can specify an arbitrary value directly in Tailwind: 

`max-w-[240px]`

This way we can keep all css within className (as much as possible).  The title attribute will enable a tool tip for those on desktop.

Reply:
LOL, that was easy, first projects I've used tailwind on.  Nice feature, I've used Boostrap for 10 plus years and it does not have anything similar.

Should be good to merge now.